pkg/cmd/eventlistener/describe.go

```diff
@@ -95,8 +104,13 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
 {{ " " }}
 {{- end }}
 {{- end }}
+{{- if isTemplateRefExist $v.Template }}
+ TEMPLATE REF	APIVERSION 
+ {{ decorate "bullet" $v.Template.Ref }}	{{ $v.Template.APIVersion }}
+{{- else }}
  TEMPLATE NAME	APIVERSION
```

Comment:
Looks good to me. Only thing left might be to match that spacing for `TEMPLATE NAME` and `APIVERSION`. It's oddly long, but the spacing should be consistent.

Reply:
I am not sure about this, but both `TEMPLATE NAME   APIVERSION` and `TEMPLATE REF   APIVERSION` have one tab space.